bin/i18n/resources/dashboard/blocks.rb

```diff
@@ -0,0 +1,66 @@
+require 'fileutils'
+require 'json'
+
+require_relative '../../i18n_script_utils'
+require_relative '../../redact_restore_utils'
+
+module I18n
+  module Resources
+    module Dashboard
+      module Blocks
+        SYNC_IN_SOURCE_FILES_PATH = CDO.dir(File.join(DASHBOARD_BLOCKS_DIR, '**/*.json')).freeze
+        DASHBOARD_I18N_FILE_PATH = CDO.dir(File.join(DASHBOARD_I18N_DIR, 'blocks.en.yml')).freeze
+        SYNC_IN_RESULT_FILE_PATH = CDO.dir(File.join(I18N_SOURCE_DASHBOARD_DIR, 'blocks.yml')).freeze
```

Comment:
I'm not sure I understand you. May you please provide more context?

Reply:
For example, where is the variable `SYNC_IN_SOURCE_FILES_PATH` used? on the `prepare` method.

Not sure it needs to be a global variable.

Let me know if this make sense or need more clarification

Reply:
How exactly are you proposing to define it only for the `.prepare` method, and why?

1. `SYNC_IN_SOURCE_FILES_PATH` constant is planning to be used in `sync_in` and `sync_out` methods.
2. It's not a "global variable", it's a constant that is defined and used within `I18n::Resources::Dashboard::Blocks` module.

Reply:
How exactly do you propose to define it only for the `.prepare` method and why?

1. `SYNC_IN_SOURCE_FILES_PATH` constant is planning to be used in `sync_in` and `sync_out` methods.
2. It's not a "global variable", it's a constant that is defined and used within `I18n::Resources::Dashboard::Blocks` module.

Reply:
oh, it is just moving code.

With the current implementation, the variable was used in one spot. Which is why I suggested this. If this is gonna change, then I get it.


Personally, I would prefer to keep the variables within the scope they are being used (even if the code will evolve later). Not a blocker since you are actively working on this